src/task.ts

```diff
@@ -166,9 +166,12 @@ export abstract class CompoundTask extends DFTask {
         children.map((c) => (c.parent = this));
         this.firstError = undefined;
 
-        // If the task has no children, then it's completed by definition.
+        // If the task has no children, throw an error
         if (children.length == 0) {
-            this.state = TaskState.Completed;
+            const message =
+                "Cannot initialize a CompoundTask with an empty childrens array. " +
```

Comment:
Can we delete this line? I feel like both lines say the same thing and I like the bottom one better

Reply:
Done